src/compute/pre_compute_app.rs

```diff
@@ -355,6 +358,24 @@ mod tests {
         }
     }
 
+    fn start_container() -> (Container<GenericImage>, String, String) {
+        let container = GenericImage::new("kennethreitz/httpbin", "latest")
+            .with_exposed_port(80.tcp())
+            .with_wait_for(WaitFor::message_on_stderr("Listening at"))
+            .with_network("bridge")
+            .with_env_var("DEBUG", "1")
+            .start()
+            .expect("Failed to start Httpbin");
+        let port = container
+            .get_host_port_ipv4(80)
+            .expect("Could not get host port");
+
+        let json_url = format!("http://127.0.0.1:{port}/json");
+        let xml_url = format!("http://127.0.0.1:{port}/xml");
+
+        (container, json_url, xml_url)
```

Comment:
Why do you return the `container` if you need to call it `_container` in calling methods because it is unused there ?